### PR TITLE
Hear the voice of the chosen deity in your head.

### PR DESCRIPTION
**Pull request by xxalpha**
You hear the voice of the badmin in your head...

### CODE REVIEW

File: code/modules/admin/verbs/randomverbs.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	if(usr)
 		if (usr.client)
 			if(usr.client.holder)
-				M << "\bold You hear a voice in your head... \italic [msg]"
+				M << "\bold You hear [ticker.Bible_deity_name ? "the voice of " + ticker.Bible_deity_name : "a voice"] in your head... \italic [msg]"
```
Comment (d3athrow) [inline code/modules/admin/verbs/randomverbs.dm]: >leaving the text macros

For shame

**Comment by Iamgoofball** [comment]
:+1:

File: code/modules/admin/verbs/randomverbs.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/admin/verbs/randomverbs.dm]: do we even have italics and bold spanclasses

Comment (RemieRichards) [inline code/modules/admin/verbs/randomverbs.dm]: @Iamgoofball <i>Just use</i><b>the HTML</b>

Comment (xxalpha) [inline code/modules/admin/verbs/randomverbs.dm]: Oh, should I change that?

Comment (RemieRichards) [inline code/modules/admin/verbs/randomverbs.dm]: Please do, \red and \blue are fairly common, and hated.
\bold and \italics are rare as fuck, and should be hated more so.

Comment (xxalpha) [inline code/modules/admin/verbs/randomverbs.dm]: Done it.

**Comment by RemieRichards** [comment]
and squashed the commits too without being asked? :+1: great work.

**Comment by vista-** [comment]
You hear the voice of ONE DAY WHILE ANDY WAS MASTURBATING in your head...

**Comment by GunHog** [comment]
Does it happen for the Chaplain only, or the entire crew?

**Comment by paprka** [comment]
Should make the chaplain the only one able to pray as well while you're at it so admins might take prayers more seriously

**Comment by dannno** [comment]
>Should make the chaplain the only one able to pray as well while you're at it

no

**Comment by paprka** [comment]
...Why not?

**Comment by dannno** [comment]
That's a big change that also is shit

**Comment by paprka** [comment]
But why

**Comment by GunHog** [comment]
Because players asking for things will just use adminhelp instead.

**Comment by paprka** [comment]
Players shouldn't be asking for admin spawns unless they're the chaplain though, that's like his whole purpose, and it would give more meaning to prayers which typically go unanswered or cookies are spawned

**Comment by xxalpha** [comment]
I highly disagree with only chaplains being able to pray, so I am not doing that.

**Comment by GunHog** [comment]
Admins are people. You cannot expect them to care about the player's status as Chaplain. I imagine admins more likely respond to people who have arranged complex and/or amusing things prior to the prayer, or offer up something valuable in exchange (traitors offering their TC for example).

**Comment by paprka** [comment]
I don't get why that can't be said in ahelp or be converted to a different system. Praying is IC and praying to have your TC traded seems retarded

**Comment by dannno** [comment]
There's a discussion thread for this already

**Comment by Niknakflak** [comment]
Paprka that is a seperate idealogy not for this pr

**Comment by paprka** [comment]
thanks for contributing

**Comment by dannno** [comment]
:laughing:

**Comment by SuperSayu** [comment]
This looks like it will break if there is no chaplain

**Comment by phil235** [comment]
Tested it. I works fine even if there's no chaplain.

**Comment by xxalpha** [comment]
How would it break if I have a thing to check for the name there?

**Comment by SuperSayu** [comment]
sorry, I haven't been getting much sleep lately.  Guess I missed it.

**Comment by MrStonedOne** [comment]
Any body can pray (irl), i see no ooc or game balance reason for them to not be able to pray.

You know @paprka, there are other ways of fixing things that don't involve removing things.

(Just in case the last 7 times this conversation has came up hasn't clued you in.)

You what else you could do? Rather than removing something, you could actually fix the core issue. Its this little thing called "tweaking" (look it up).

For instance:

If people are abusing the gold in the vault to power game, instead of trying to remove the gold in the vault, you can tweak the things they use it on for power game.

If people are abusing an invisible armor item for damage protection that can't be easily detected, you can either make it noticeable, or lower the damage protection, or all sorts of suggestions!

If job default cosmetic items are breaking departmental color themes, you could, oh i don't know, move it from being a job locker item to a random cosmetic item, or maybe fix it so it doesn't break departmental color themes.

And, lastly, if there is an issue where prays by chaplains are getting drown out by the prays of other people, than you can just make the prays by chaplains more noticeable.

***
This message brought to you by Improve, Not Remove[tm]. Improve, Not Remove: Because we're worth it!(c)

**Comment by paprka** [comment]
I'm laughing my ass off

fuck off mso

**Comment by MrStonedOne** [comment]
See! You don't like me, so you want to remove me. Classical pappy.

**Comment by paprka** [comment]
ironic considering you put mod approved posting on my posts on the forum because you can't handle the bantz and want to remove me

**Comment by xxalpha** [comment]
Guys please, pray this shit so the rest of us don't have to listen.

**Comment by paprka** [comment]
:+1:

**Comment by dannno** [comment]
>See! You don't like me, so you want to remove me. Classical pappy. 

HAHAHA

**Pull request merged**